src/content/nft/index.md

```diff
@@ -108,7 +108,15 @@ NFT's have some special properties:
 In other words, if you _own_ an NFT:
 
 - You can easily prove you own it.
-  - Proving you own an NFT is very similar to proving you have ETH in your account
+  - Proving you own an NFT is very similar to proving you have ETH in your account.
+  - For example, lets say you pay for an NFT. What you are getting is the right to transfer your token to your digital wallet.
+  - The token proves that your copy of the digital file is the original.
+  - Your private crypto key is proof of ownership of the original.
+  - The content creator's public crypto key serves as a certificate of authenticity for that particular digital artifact.
+    - This public and private key pair is what primarily determines the value of any NFT token.
```

Comment:
This is a little confusing to me... Looks like you're trying to explain that the creators public key is essentially a permanent part of the tokens history, and can be used to demonstrate that the token you hold was in fact created by a particular individual, thus contributing to its market value (vs a counterfeit)... does that sound right? We may just want to adjust the wording here to be a little more clear

Reply:
Yup I agree! Made the necessary changes